incubator/etcd/Chart.yaml

```diff
@@ -9,3 +9,5 @@ sources:
 maintainers:
 - name: Lachlan Evenson
   email: lachlan@deis.com
+- name: Alex Knol
```

Comment:
For the maintainer names we now require github usernames. That's so we can more easily contact chart maintainers in their duties at maintainer. Can you please use your github username.

If you feel up to it, you could also update @lachie83 to his username and update or remove his email address as that one no longer works.